modules/learn/pages/security/roles.adoc

```diff
@@ -1,2340 +1,2330 @@
+
+[#table_security_admin_local_role,cols="1,2,2,hrows=2"]
 |===
-5+^| Role: Local User Security Admin (`security_admin_local`)
+3+^| Role: Local User Admin (`user_admin_local`)
+
+h| Resource
+h| Permissions
+h| Restrictions
+
+| *Servers* 
+| View statistics
+| Add, failover, remove, modify services, rebalance
+
+| *Buckets*
+| List buckets, scopes, and collections
+| Create, drop, edit settings, read or write data
+
+| *Backup*
+| None
+| All
+
+| *XDCR* 
+| All
+| None
+
+| *Security* 
+| Add, delete, edit local users. 
+Can add, delete, and edit groups.
+| Cannot grant Full Admin, Read-Only Admin, Local or External User Admin, or Security Admin roles to users or groups.
+Cannot access non-user and group security resources.
 
-.2+^h| Resources
-4+^h| Privileges
+| *Settings*
+| View
+| Change
 
-^h| *Read*
-^h| *Write*
-^h| *Execute*
-^h| *Manage*
+| *Logs*
+| View
+| Collect Information
 
-^| Cluster
-^| image:introduction/yes.png[]
-^| image:introduction/no.png[]
-^| image:introduction/no.png[]
-^| image:introduction/no.png[]
+| *Query*
+| None
+| All
 
-^| UI (except Local User and Group Security)
-^| image:introduction/yes.png[]
-^| image:introduction/no.png[]
-^| image:introduction/no.png[]
-^| image:introduction/no.png[]
+| *Search*
+| None
+| All
 
-^| Local User and Group Security (including UI)
-^| image:introduction/yes.png[]
-^| image:introduction/yes.png[]
-^| image:introduction/yes.png[]
-^| image:introduction/yes.png[]
+| *Analytics*
+| None
+| All
+
+| *Eventing*
+| None
+| All
+
+| *Views*
+| None
+| All
+
+|===
+
+
+[#eventing-full-admin]
+=== Eventing Full Admin
+
+The Eventing Full Admin role lets a user create and manage eventing functions as well as other administration tasks.
+
+The role lets the user log into the Couchbase Server Web Console.
+
+[#table_eventing_admin_role,cols="1,2,2,hrows=2"]
+|===
+3+^|  Role: Eventing Full Admin (`eventing_admin`)
+
+h| Resource
+h| Permissions
+h| Restrictions
+
+| *Servers* 
+| View configuration and statistics
+| Cannot add, failover, remove, modify services, rebalance
+
+| *Buckets*
+| List buckets, scopes, and collections. 
+Can create, compact, and drop buckets, scopes, and collections.
+Can read and write data in buckets.
+| None
+
+| *Backup*
+| List repositories and plans
+| Cannot add or edit repositories or plans 
+
+| *XDCR* 
+| List outgoing replications 
+| Cannot list incoming replications or remote clusters.
+Cannot add or edit replications.
+
+| *Security*
+| None
+| All
+
+| *Settings*
+| View all settings and load sample buckets
+| Cannot edit settings
+
+| *Logs*
+| View logs
+| Cannot collect information
+
+| *Indexes*
+| All
+| None
+
+| *Query*
+| All
+| None
+
+| *Search*
+| All
+| None
+
+| *Analytics*
+| All   
+| None
+
+| *Eventing*
+| All
+| None
+
+| *Views*
+| All  
+| None
+
+|===
+
+
+[#backup-full-admin]
+=== Backup Full Admin
+
+The Backup Full Admin role lets the user administer backup-related tasks as well as other aspects of Couchbase Server.
+
```

Comment:
The backup_admin role by itself cannot backup/restore users.  To be able to backup/restore users, the following additional roles must be assigned to the user -- user_admin_local, user _admin_external